### ISSUE TITLE
Leukocyte casts

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5047]
Definition:Presence of white blood cell casts
Modified definition: Presence of leukocyte casts (cylindrical structures produced by the kidney in certain disease states) in the urine.

**Comment by pnrobinson** [comment]
This is synonymous with https://hpo.jax.org/app/browse/term/HP:0012616 (Leukocyte cylindruria)

I will change the primary label to Leukocyte casts and retain Leukocyte cylindruria, OK?

@avizrosenberg
@beckysteck
@yongqunh

**Comment by yongqunh** [comment]
That's good for me.

**Comment by avizrosenberg** [comment]
Cylinduria refers to the identification of casts in the urine. When we refer to casts in the tissue this should be discrete from the urinary microscopic findings.

**Comment by pnrobinson** [comment]
@avizrosenberg Are you saying we should have separate terms?
e.g. Erythrocyte cylindruria for urine findings and Erythrocyte casts for renal histology? If so, what could we say to make sure that the labels are clear ("casts" can also refer to the urine finding).

**Comment by pnrobinson** [comment]
Adding this with the label

Intratubular leukocyte casts
and definition:
Casts that contain white blood cells and are located within the tubuli of the kidney.

@avizrosenberg  Please reopen if this requires revision.

**Comment by avizrosenberg** [comment]
@pnrobinson defining as intratubular should be sufficient to distinguish from urinary.
